docs/getting-started-own-model.md

```diff
@@ -2,8 +2,21 @@
 
 This guide will show you how to put your own machine learning model in a Docker image using Cog. If you haven't got a model to try out, you'll want to follow the [main getting started guide](getting-started.md).
 
+## Prerequisites
+
+- **A trained model**. If you don't already have your own trained model, you can use one from [https://github.com/replicate/cog-examples](replicate/cog-examples).
```

Comment:
Might be worth a nod to "Get started with an example model" in `getting-started.md` here?

Reply:
Gotcha. Removed that line from the prereqs because it's a bit redundant given the context.